lib/assessment.md

```diff
@@ -82,21 +100,16 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+specialDiv will have red color text in cursive font-family.
```

Comment:
Close. Font family is correct, but color is incorrect. The specialDiv isn't actually a member of the `things` class, it just sits inside an element that is. Thus, `color: red;` is an _inherited_ property.